exercises/accumulate/examples/src/Accumulate.purs

```diff
@@ -0,0 +1,14 @@
+module Accumulate
+  ( accumulate
+  ) where
+
+import Data.List (List(Nil), (:), reverse)
+
+
+accumulate :: forall a b . (a -> b) -> List a -> List b
+accumulate f values = accumulate' f Nil values
+
+
+accumulate' :: forall a b . (a -> b) -> List b -> List a -> List b
```

Comment:
Sounds good.

Is there a convention about when to use `where` and when to use `let`? I've been using `let` as I find it a bit easier to read with only one style.